pkg/granted/completion.go

```diff
@@ -39,8 +46,13 @@ var CompletionCommand = cli.Command{
 			// Try create a file
 			f, err := os.Create(executableDir)
 
+			// If that fails, try to open an existing file
 			if err != nil {
-				fmt.Fprintln(os.Stderr, "Something went wrong when saving fish autocompletions: "+err.Error())
+
+				f, err = os.Open(executableDir)
+				if err != nil {
+					fmt.Fprintln(os.Stderr, "Something went wrong when saving fish autocompletions: "+err.Error())
+				}
```

Comment:
These lines are important, ensures the file is overwritten if it already exists

Reply:
@jordiup I think you can use a builtin method in Go to open/create then write to a file

os.WriteFile(executableDir, []byte(output), 0600) 

This should handle everything for you and you wont need to create or open yourself